ios/kmbuild.sh

```diff
@@ -119,8 +119,9 @@ DEFAULT_LM_ID="nrc.en.mtnt"
 APP_BUNDLE_PATH=$BUILD_PATH/${CONFIG}-iphoneos/Keyman.app
 KEYBOARD_BUNDLE_PATH=$BUILD_PATH/${CONFIG}-iphoneos/SWKeyboard.appex
 ARCHIVE_PATH=$BUILD_PATH/${CONFIG}-iphoneos/Keyman.xcarchive
-FRAMEWORK_PATH_UNIVERSAL=$BUILD_PATH/$CONFIG-universal/KeymanEngine.framework
-FRAMEWORK_PATH_IOS=$BUILD_PATH/$CONFIG-iphoneos/KeymanEngine.framework
+
+# Engine library build path
+KEYMAN_XCFRAMEWORK=$BUILD_PATH/$CONFIG/KeymanEngine.xcframework
```

Comment:
Fortunately, the new XCFramework setup is remarkably transparent:

![Screen Shot 2021-05-19 at 3 58 50 PM](https://user-images.githubusercontent.com/25213402/118785564-2c815280-b8bb-11eb-9a39-c7d884d8b391.png)

The two folders in the second column correspond to the two different build targets, with the final XCFramework replacing the old "universal" fat-framework:
- `ios-arm64_armv7` - the app for actual iOS devices
- `ios-arm64_i386_x86_64-simulator` - the simulator version

Reply:
Also, please note the paths defined here:
- For release builds, the `.xcframework` will be found at `../Release/KeymanEngine.xcframework` **_relative_** to the old product's path.